benefits/core/templates/core/help.html

```diff
@@ -116,19 +116,51 @@ <h2 class="h2-sm pt-4 pt-lg-8" id="{{ flow_help.id }}">{{ flow_help.headline }}<
             <p class="pt-2 pt-lg-4">{{ flow_help.text|safe }}</p>
           {% endfor %}
         {% endif %}
-        <h2 class="h2-sm pt-4 pt-lg-8" id="littlepay">{% translate "What is Littlepay?" %}</h2>
-        <p class="pt-2 pt-lg-4">
-          {% blocktranslate trimmed %}
-            Our payment partner, Littlepay, is a secure, end-to-end payment processing platform that allows us to connect your
-            transit benefit to the contactless card of your choice.
-          {% endblocktranslate %}
-        </p>
-        <p class="pt-3 pt-lg-4">
-          {% blocktranslate trimmed %}
-            For more information on Littlepay, please visit the
-            <a href="https://littlepay.com" target="_blank" rel="noopener noreferrer">Littlepay website</a>.
-          {% endblocktranslate %}
-        </p>
+
+        {% if agency and agency.littlepay_config %}
+          <h2 class="h2-sm pt-4 pt-lg-8" id="littlepay">{% translate "What is Littlepay?" %}</h2>
+          <p class="pt-2 pt-lg-4">
+            {% blocktranslate trimmed %}
+              Our payment partner, Littlepay, is a secure, end-to-end payment processing platform that allows us to connect your
+              transit benefit to the contactless card of your choice.
+            {% endblocktranslate %}
+          </p>
+          <p class="pt-3 pt-lg-4">
+            {% blocktranslate trimmed %}
+              For more information on Littlepay, please visit the
+              <a href="https://littlepay.com" target="_blank" rel="noopener noreferrer">Littlepay website</a>.
+            {% endblocktranslate %}
+          </p>
+        {% elif agency.switchio_config %}
```

Comment:
This should also check `agency`, like the `if` condition, right? 

```
{% elif agency and agency.switchio_config %}
```

Reply:
I think the `else` is fine. If no agency or no config is present, it makes sense to show the full string / all options.